### ISSUE TITLE
YouTube parameters

### ISSUE THREAD

**Issue by joppehoekstra** [ueberdosis/tiptap#3270]
### What problem are you facing?

I would like to set [parameters](https://developers.google.com/youtube/player_parameters#Parameters) for an embedded YouTube video. This is currently not possible.

### What’s the solution you would like to see?

Add an option to `Youtube.configure()` to set [parameters](https://developers.google.com/youtube/player_parameters#Parameters) for the embed url, like this: 
```
Youtube.configure({
  parameters: [{autoplay: 1}, {modestbranding: 1}]
})
```
Which should result in the following url:
`https://www.youtube.com/embed/VIDEO_ID?autoplay=1&modestbranding=1`

### What alternatives did you consider?

Set these parameters with the `setYoutubeVideo()` command. This would be useful if you would want to set different parameters per video (though this is not needed for my personal use case).

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by LuchoCateura** [comment]
https://github.com/ueberdosis/tiptap/pull/3307

**Comment by bdbch** [comment]
Should be included via #3307

**Comment by Harm-Nullix** [comment]
I wouldn't say this is included by #3307 ,
It does make it able to set it in the config of the extension, but not by each video as I read the commits.
